doc/03-Configuration.md

```diff
@@ -0,0 +1,70 @@
+# Configuration
+
+![Icinga Notifications Web Preview](res/notifications-preview.png)
+
+If Icinga Web has been installed but not yet set up, please visit Icinga Web and follow the web-based setup wizard.
+For Icinga Web setups already running, log in to Icinga Web with a privileged user and follow the steps below to
+configure Icinga Notifications Web:
+
+<!-- {% if not icingaDocs %} -->
+
+## Module Activation
+
+If you just installed the module, do not forget to activate it on your Icinga Web instance(s) by using your
+preferred way:
+
+- Use Icinga Web's command-line interface on the webserver(s) and execute `icingacli module enable notifications`.
+- Visit Icinga Web, log in as a privileged user and activate the module under `Configuration →
+  Modules → notifications` by switching the state from `disabled` to `enabled`.
+
+<!-- {% endif %} -->
+
+## Database Configuration
+
+Connection configuration for the database, which both,
+[Icinga Notifications](https://github.com/Icinga/icinga-notifications) and [Icinga Notifications Web](https://github.com/Icinga/icinga-notifications-web), use.
+
+!!! tip
+    If not already done, initialize your database by following the [instructions](https://icinga.com/docs/icinga-notifications/latest/doc/02-Installation#setting-up-the-database).
+
+1. Create a new resource for the Icinga Notifications database via the `Configuration → Application → Resources` menu.
+2. Configure the resource you just created as the database connection for the Icinga Notifications Web module using the
+   `Configuration → Modules → notifications → Database` menu.
+
+## Channels Configuration
+
+As the Icinga Notifications daemon notifies contacts in case of events and incidents, you need to configure appropriate 
+communication channels.
+
+The currently supported channels can be found [here](01-About.md#available-channels).
+
+They can be configured through `Configuration → Modules → notifications → Channels` and the credentials to be supplied 
+might differ depending on the channel type.
+
+You need to configure at least one valid communication channel to be able to supply your contacts with notifications.
+
+## Sources Configuration
+
+The notifications module operates on data fed by miscellaneous sources and is therefore not restricted to Icinga 2 only.
+Consult the source specific documentation on how to integrate such.
+
+You need to provide at least one valid source for this module to function properly.
+
+### Adding an Icinga 2 source
+
+!!! tip
+    If there is no API user with the required permissions yet, read through [Icinga Web's API documentation](https://icinga.com/docs/icinga-web/latest/modules/monitoring/doc/05-Command-Transports/#use-the-icinga-2-api).
+
+    The API user needs the following [permissions](https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/#overview):
+
+    - `events/*`
+
+    - `objects/query/*`
```

Comment:
Just noticed several things here:

* `!!! tip` must go on its own line and be followed by an empty line, otherwise this looks odd on Github
* Don't link to Icinga Web's documentation, this not only is the monitoring module (legacy) documentation, but also irritates the user probably to that extent, that they think setting up commandtransports there might be required
* The link to the API documentation isn't rendered on Github (Though I have no idea how to prevent this, while still showing this inside the tip)
* The empty lines around the bullet points also look odd, probably not only on Github

Reply:
> !!! tip must go on its own line and be followed by an empty line, otherwise this looks odd on Github

![image](https://github.com/Icinga/icinga-notifications-web/assets/146938323/a541cae0-0acc-4217-a029-605d82f37003)

The docs are optimized to be displayed through MKDocs-Material as that's the format that is going to land on our official documentation site.

> Don't link to Icinga Web's documentation, this not only is the monitoring module (legacy) documentation, but also irritates the user probably to that extent, that they think setting up commandtransports there might be required

Well, that's the current place where we documented the API feature. IcingaDB has an own API, but that one differs to the main one and it might confuse a potential reader even more, especiall since IcingaDB is not listed as a requirement.

> The empty lines around the bullet points also look odd, probably not only on Github

The empty lines are on purpose, as MKDocs renders admonitions like standard Markdown. Which means a single line break gets lost and this results in an overclustered paragraph:
![image](https://github.com/Icinga/icinga-notifications-web/assets/146938323/62a18e1c-2777-4fb0-9e1a-2c7704217fd2)

Reply:
> The docs are optimized to be displayed through MKDocs-Material as that's the format in which it is going to land on our official documentation site.

You don't have to tell me that. But it should look somewhat ok on Github still. MKDocs should be fine with that empty line I'm talking about, since it works in Icinga DB Web's documentation as well.

> Well, that's the current place where we documented the API feature. IcingaDB has an own API, but that one differs from the main one and it might confuse a potential reader even more, especially since IcingaDB is not listed as a requirement.

Of course, Icinga DB's commandtransport documentation would be wrong either. Show an example or link to https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/

>  as MKDocs renders admonitions like standard Markdown

Seemingly not, as otherwise they would be shown in a list. Use two trailing spaces then, if that's honored by MKDocs...

Reply:
> You don't have to tell me that. But it should look somewhat ok on Github still. MKDocs should be fine with that empty line I'm talking about, since it works in Icinga DB Web's documentation as well.

I know and I've added it, just wanted to make clear what the intentions where in terms of file formatting / syntax usage.

> Of course, Icinga DB's commandtransport documentation would be wrong either. Show an example or link to https://icinga.com/docs/icinga-2/latest/doc/12-icinga2-api/

To be fair, I thought I linked to that document. Not sure how I ended up putting the one from icingaweb in there. Probably because I had opened up both of them next to each other...

> Seemingly not, as otherwise they would be shown in a list. Use two trailing spaces then, if that's honored by MKDocs...

True, just double-checked it with python-markdown. Seems to be related to how MKDocs Material parses paragraphs. Apparently they remove line breaks in the string and just cut the paragraphs when a line break on an otherwise empty line gets parsed.